dataflow/manual/content.tex

```diff
@@ -1734,6 +1734,35 @@ \section{Example: Live Variable}
 
 \flow{LiveSimple}{.33}{1}{Simple sequential program to illustrate live variable.  Intermediate analysis results are shown.}
 
+\section{Example: Very Busy Expression Analysis}
+
+Unlike the live variable analysis which combines abstract states from successors,
+very busy expression analysis is a backward analysis that intersects abstract states from successors.
+An expression is very busy if no matter what path is taken, the expression is used before any the variables
```

Comment:
```suggestion
An expression is very busy if no matter what path is taken, the expression is used before any of the variables
```

Reply:
Thanks!

Reply:
sure, I have already changed it. I just noticed Alex made some changes to the cfg-visualize code, so I haven't pushed the changes yet.